blazar_dashboard/content/leases/widgets.py

```diff
@@ -30,7 +30,7 @@ def get_context(self, name, value, attrs=None):
             'value': value,
         }}
 
-    def render(self, name, value, attrs=None):
+    def render(self, name, value, attrs=None, renderer=None):
```

Comment:
Is this kwarg used, or was this an abandoned idea?

Reply:
This is a Django 2.1 upgrade requirement: https://docs.djangoproject.com/en/2.1/releases/2.1/#features-removed-in-2-1

My devstack installation is running Django 2.2. I've reverted this change in the latest commit, and the upgrade issue can be dealt with in the future

Reply:
I believe this is a django version problem. See https://docs.djangoproject.com/en/2.1/releases/2.1/#features-removed-in-2-1

My devstack install is running Django 2.2